sections/collage.liquid

```diff
@@ -22,7 +22,7 @@
 <div class="color-{{ section.settings.color_scheme }} gradient">
   <div class="page-width{% if section.settings.heading == blank %} no-heading{% endif %} section-{{ section.id }}-padding">
     <h2 class="collage-wrapper-title">{{ section.settings.heading | escape }}</h2>
-    <div class="collage{% if section.settings.mobile_layout == 'collage' %} collage--mobile{% endif %}">
+    <div class="collage{% if section.settings.mobile_layout == 'collage' %} collage--mobile{% endif %} isolate">
```

Comment:
Usually good to have the class that we know are always there before the conditional statements.
```suggestion
    <div class="collage isolate{% if section.settings.mobile_layout == 'collage' %} collage--mobile{% endif %}">
```